node/libs/roles/src/attester/messages/l1_batch.rs

```diff
@@ -0,0 +1,122 @@
+use zksync_concurrency::time;
+
+use crate::{
+    attester::{self, AggregateSignature},
+    validator::Genesis,
+};
+
+use super::{SignedBatchMsg, Signers};
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Default, PartialOrd)]
+/// A batch number.
+pub struct BatchNumber(pub u64);
+
+impl BatchNumber {
+    /// Increment the batch number.
+    pub fn next_batch_number(&self) -> BatchNumber {
```

Comment:
just "next()" - "batch_number" is redundant.

Reply:
Yeah, I totally agree. It's weird because `next()` was my first choice, and I remember Clippy saying that it could clash with an `iter()` implementation for `BatchNumber`, but it's not raising any issues anymore, so I assume it's okay now.